### PR TITLE
refactor tokengen and txgen to avoid go mod import

### PR DESCRIPTION
**Pull request by adecaro**
This PR does the following:
- It removes txgen ad command line tool
- It moves the bulk of the txgen's code under integration/nwo
- It moves the tokengen's cobra commands under `cmd/tokegen`
- It simplifies go mod dependencies

### CODE REVIEW

**Comment by arner** [comment]
The approach is at least consistent with tokengen; the cmd directory is the entrypoint for code that is in the integration folder. There is some inconstency with certifier-keygen and pp though (and `tokengen version` - which all have some part in token/core). It's also not that pretty that all the commands depend on the integration package: for instance `tokengen` is also used for production... but then it imports integration because it also does artifactgen which is 100% for testing. I guess that was the reason @alexandrosfilios had the txgen cmd in the integration folder; to make the difference clear.

Could there be a more clear or consistent approach? If we are ok with having commands purely for testing in the cmd folder (which is fine with me), shouldn't we then have **all** the cobra stuff inside `cmd` folder? In the spirit of clean architecture and separation of concerns, splitting the presentation layer from the logic. Although if there's code that's _only_ used by the cli tools, we might as well put that in the cmd folder too... What do you think?

**Comment by adecaro** [comment]
@arner , good point. Let me try to refactor the code accordingly. If a tool is used only for testing, we can have it positioned under `integration/nwo/cmd`. What do you think?

File: integration/nwo/txgen/service/user/rest/user.go [hunk 1]
```diff
@@ -17,11 +17,11 @@ import (
 	"strings"
 	"time"
 
-	"github.com/hyperledger-labs/fabric-token-sdk/txgen/model"
-	"github.com/hyperledger-labs/fabric-token-sdk/txgen/model/api"
-	c "github.com/hyperledger-labs/fabric-token-sdk/txgen/model/constants"
-	"github.com/hyperledger-labs/fabric-token-sdk/txgen/service/logging"
-	"github.com/hyperledger-labs/fabric-token-sdk/txgen/service/metrics"
+	"github.com/hyperledger-labs/fabric-token-sdk/integration/nwo/txgen/model"
+	api2 "github.com/hyperledger-labs/fabric-token-sdk/integration/nwo/txgen/model/api"
```
[the remark this answers is not in the document]
Comment (adecaro) [reply integration/nwo/txgen/service/user/rest/user.go]: 👍

File: integration/nwo/txgen/executor.go [hunk 2]
```diff
@@ -10,13 +10,13 @@ import (
...
-	"github.com/hyperledger-labs/fabric-token-sdk/txgen/service/logging"
-	"github.com/hyperledger-labs/fabric-token-sdk/txgen/service/metrics"
-	"github.com/hyperledger-labs/fabric-token-sdk/txgen/service/runner"
-	"github.com/hyperledger-labs/fabric-token-sdk/txgen/service/user"
-	"github.com/hyperledger-labs/fabric-token-sdk/txgen/service/user/rest"
+	"github.com/hyperledger-labs/fabric-token-sdk/integration/nwo/txgen/model"
+	"github.com/hyperledger-labs/fabric-token-sdk/integration/nwo/txgen/model/api"
+	"github.com/hyperledger-labs/fabric-token-sdk/integration/nwo/txgen/service/logging"
+	metrics3 "github.com/hyperledger-labs/fabric-token-sdk/integration/nwo/txgen/service/metrics"
+	runner2 "github.com/hyperledger-labs/fabric-token-sdk/integration/nwo/txgen/service/runner"
```
[the remark this answers is not in the document]
Comment (adecaro) [reply integration/nwo/txgen/executor.go]: 👍

**Comment by adecaro** [comment]
@arner , all comments addressed. Thanks :)

**Review by alexandrosfilios — COMMENTED** [review]
LGTM

**Review by alexandrosfilios — APPROVED** [review]
LGTM

**Pull request merged**